src/hayhooks/server/pipelines/utils.py

```diff
@@ -522,17 +527,17 @@ def _setup_hybrid_streaming_callbacks_for_pipeline(
 
     if streaming_components == "all":
         components_to_stream = all_streaming_components
-        log.trace("Hybrid streaming enabled for all components via 'all' keyword")
+        log.trace("Hybrid streaming enabled for all components via 'all' keyword", components=components_to_stream)
     elif streaming_components is None:
         if all_streaming_components:
             components_to_stream = [all_streaming_components[-1]]
-            log.trace(f"Hybrid streaming enabled for last component only: {all_streaming_components[-1][1]}")
+            log.trace("Hybrid streaming enabled for last component only", component=all_streaming_components[-1][1])
     elif isinstance(streaming_components, list):
         enabled_component_names = set(streaming_components)
         for component, component_name in all_streaming_components:
             if component_name in enabled_component_names:
                 components_to_stream.append((component, component_name))
-        log.trace(f"Hybrid streaming enabled for components: {[name for _, name in components_to_stream]}")
+        log.trace("Hybrid streaming enabled for components", components=[name for _, name in components_to_stream])
```

Comment:
so to confirm in this case we don't want to put the component names in the log message but just attach them to the log itself?